deploy/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "bucket_name" {
+  default = "answerking-bucket-react"
+}
+
+variable "ssl_certificate_arn" {
+  description = "The certificate ARN for the provided domain. Be aware that for cloud front the certicate needs to be available in us-east-1."
```

Comment:
description probably needs updating to `eu-west-2`?

Reply:
unless cloudfront just always uses `us-east-1` or something

Reply:
Yes cloudfront has to be pointed to us-east-1